update/sync/coordinator.go

```diff
@@ -61,23 +64,31 @@ func NewSyncState(args ArgsNewSyncState) (*syncState, error) {
 
 // SyncAllState gets an epoch number and will sync the complete data for that epoch start metablock
 func (ss *syncState) SyncAllState(epoch uint32) error {
-
+	ctxDisplay, cancelDisplay := context.WithCancel(context.Background())
+	go displayStatusMessage(fmt.Sprintf("syncing un-finished meta headers for epoch %d", epoch), ctxDisplay)
 	ss.syncingEpoch = epoch
 	err := ss.headers.SyncUnFinishedMetaHeaders(epoch)
+	cancelDisplay()
 	if err != nil {
-		return err
+		return fmt.Errorf("%w in syncState.SyncAllState - SyncUnFinishedMetaHeaders", err)
 	}
 
+	ctxDisplay, cancelDisplay = context.WithCancel(context.Background())
+	go displayStatusMessage("syncing epoch start metablock", ctxDisplay)
```

Comment:
You also can add epoch like in the one from line 68

Reply:
done